tests/suites/0_stateless/20+_others/20_0007_http_clickhouse_handler.sh

```diff
@@ -60,4 +60,4 @@ curl -s  -u 'root:' -XPOST "http://localhost:${QUERY_CLICKHOUSE_HTTP_HANDLER_POR
 curl -s  -u 'root:' -XPOST "http://localhost:${QUERY_CLICKHOUSE_HTTP_HANDLER_PORT}" -d "drop table a"
 
 
-# rm /tmp/*.txt
+rm /tmp/*.txt
```

Comment:
Seems dangerous to me...

Reply:
I think so too. We delete it one by one.